src/plone/restapi/deserializer/blocks.py

```diff
@@ -32,42 +29,6 @@ def iterate_children(value):
             queue.extend(child["children"] or [])
 
 
-def path2uid(context, link):
```

Comment:
@sneridagh nitpicking: I usually prefer to keep util functions in the location where they are used. As long as a util function is only used in one location, moving it to its own file just makes it harder to understand what goes on.

Reply:
For the record, I moved them to its own module due to circular import issues.

Reply:
I'd argue that a Python library also has to consider the API it exposes as public. Would an import such as `from plone.restapi.blocks import path2uid` make sense? Not much. If the `path2uid` would be an internal util used only in the blocks.py module, then it should also be marked as internal, like `_path2uid`.